docs/python/ReadMeOV.rst

```diff
@@ -0,0 +1,79 @@
+OpenVINO™ Execution Provider for ONNX Runtime
+===============================================
+
+-  `OpenVINO™ Execution Provider for ONNX Runtime <https://github.com/microsoft/onnxruntime/>`_. is a product designed for OnnxRuntime developers who want to get started with OnnxRuntime in their inferencing applications. This product delivers  `OpenVINO™ <https://software.intel.com/content/www/us/en/develop/tools/openvino-toolkit.html>`_. inline optimizations which enhance inferencing performance with minimal code modifications. OpenVINO™ Execution Provider for ONNX Runtime accelerates inference across many  `AI models <https://github.com/onnx/models>`_. on a variety of Intel silicon such as:
+- Intel CPUs
+- Intel integrated GPUs
+- Intel Movidius™ Vision Processing Units - referred to as VPU
+- Intel Vision Accelerator Design with 8 Intel Movidius™ MyriadX VPUs - referred to as VAD-M or HDDL
+
+Installation
+------------
+
+Requirements
+^^^^^^^^^^^^
+
+- Ubuntu 18.04, macOS 11.2.3 or Windows 10 - 64 bit
+- Python* 3.7, 3.8 or 3.9
+
+Windows release supports only Python3.9 
+
+This **OpenVINO™ Execution Provider for ONNX Runtime** package comes with pre-built libraries of OpenVINO™ version 2022.1.0 meaning you do not have to install OpenVINO™ separately.
+This package supports:
```

Comment:
This list's format doesn't reflect in the final doc, it might need empty lines in between.